From 32f1ba8e30857767e89a76e6fed590dd53939d16 Mon Sep 17 00:00:00 2001
From: Robbie Harwood <rharwood@redhat.com>
Date: Wed, 11 Nov 2015 15:53:58 -0500
Subject: [PATCH] Document existing kdc option kdc_max_dgram_reply_size

This was introduced as part of the documentation system overhaul to ReST, so
this is just the text lifted from that.
---
 src/config-files/kdc.conf.M | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/config-files/kdc.conf.M b/src/config-files/kdc.conf.M
index b47c251..50d42c7 100644
--- a/src/config-files/kdc.conf.M
+++ b/src/config-files/kdc.conf.M
@@ -75,6 +75,9 @@ flag to
 If this relation is not specified, the compiled-in default of
 .I none
 is used.
+.IP kdc_max_dgram_reply_size
+Specifies the maximum packet size that can be sent over UDP.  The default
+value is 4096 bytes.
 
 .SH REALMS SECTION
 Each tag in the
-- 
2.6.2

